pkg/proc/core/core.go

```diff
@@ -243,9 +249,9 @@ func (p *Process) EntryPoint() (uint64, error) {
 	return p.entryPoint, nil
 }
 
-// writeBreakpoint is a noop function since you
+// WriteBreakpoint is a noop function since you
 // cannot write breakpoints into core files.
-func (p *Process) writeBreakpoint(addr uint64) (file string, line int, fn *proc.Function, originalData []byte, err error) {
+func (p *Process) WriteBreakpoint(addr uint64) (file string, line int, fn *proc.Function, originalData []byte, err error) {
```

Comment:
writeBreakpoint really shouldn't be an exported function, since it can't be called safely by anyone except the breakpoint map.

Reply:
In my opinion I don't think anything in `pkg/proc` should even know about breakpoints after this refactor.